### PR TITLE
Gloria downloader

### PR DESCRIPTION
**Pull request by hazimhussein**
This branch contains the implementation of the GLORIA downloader and the related documentation

### CODE REVIEW

**Comment by konstantinstadler** [comment]
seems like you did not format (black and isort) the pull request. You can have a look into the format_and_test.sh file on how to do that. This is a linux file, but you can copy paste the commands in cml or write your own bat file (actually, you could translate/make a format_and_test.bat similar to the sh file for convinience, just push it with a pull request when you see that it is working)

File: tests/test_util.py [hunk 1]
```diff
@@ -225,3 +227,17 @@ def test_set_block():
         full_arr = np.random.random((10, 12))
         block_arr = np.zeros((2, 2))
         mod_arr = set_block(full_arr, block_arr)
+
+
+def test_filename_from_url():
+    letters = np.array(list(string.printable))
+    fake = Faker()
```
Comment (konstantinstadler) [inline tests/test_util.py:234]: Good idea with Faker!
Great job, but please add also 2-3 string urls to the test function - we also have that as "documentation" to see what and how the function works

File: doc/source/notebooks/advanced_group_stressors.ipynb [hunk 2]
```diff
@@ -34,7 +34,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "wiod_folder = '/tmp/mrios/WIOD2013'"
+    "wiod_folder = \"/tmp/mrios/WIOD2013\""
```
Comment (konstantinstadler) [inline doc/source/notebooks/advanced_group_stressors.ipynb:37]: This happens on multiple places - is that something black does? Or did you do that manually?

**Review by konstantinstadler — DISMISSED** [review]

**Comment by hazimhussein** [comment]
It is 23.3.0

But black package alone didn’t work on jupyter notebook, I had to download another dependency “black-jupyter” also version 23.3.0

**Review by konstantinstadler — APPROVED** [review]
great work!

**Pull request merged**